app/test/unit/mocks/mock_nunjucks.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+/* exported Mocknunjucks */
```

Comment:
I can't change this because `MocksHelper` is just "installing" the mocks by removing the Mock prefix. And nunjucks starts with lowercase.